plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/CreateMavenBasedAppEngineStandardProject.java

```diff
@@ -84,7 +76,6 @@ protected void execute(IProgressMonitor monitor)
           project, true, loopMonitor.newChild(1));
       AppEngineStandardFacet.installAppEngineFacet(facetedProject, true /* installDependentFacets */, loopMonitor.newChild(1));
       AppEngineStandardFacet.installAllAppEngineRuntimes(facetedProject, true /* force */, loopMonitor.newChild(1));
-      PreferenceUtil.setProjectIdPreference(project, appId);
```

Comment:
We could remove the `PreferenceUtil` class (and corresponding test class) as well, as nothing else uses it.